### PR TITLE
Ppp name fix zoran

### PR DESCRIPTION
**Pull request by zoranabc201**

### CODE REVIEW

**Review by ngdeqi — COMMENTED** [review]
LGTM

File: docs/AboutUs.md [hunk 1]
```diff
@@ -3,8 +3,9 @@
 
 | Display                                             |           Name            |                Github Profile                |              Portfolio               |
 |-----------------------------------------------------|:-------------------------:|:--------------------------------------------:|:------------------------------------:|
-| ![](https://via.placeholder.com/100.png?text=Photo) |     Wilson Ng Jing An     |        [Github](https://github.com/wilsonngja)         |   [Portfolio](team/wilsonngja.md)    |
-| ![](https://via.placeholder.com/100.png?text=Photo) |      Syed Omar Zoran      |   [Github](https://github.com/zoranabc201)   |  [Portfolio](team/syedomarzoran.md)  |
+
+| ![](https://via.placeholder.com/100.png?text=Photo) |     Wilson Ng Jing An     | [Github](https://github.com/wilsonngja)      |   [Portfolio](team/wilsonngja.md)    |
+| ![](https://via.placeholder.com/100.png?text=Photo) |      Syed Omar Zoran      | [Github](https://github.com/zoranabc201)     |  [Portfolio](team/zoranabc201.md)    |
```
Comment (wilsonngja) [inline docs/AboutUs.md:8]: image

It seems like when you are editing, there's some issue with the formatting.

**Review by wilsonngja — CHANGES_REQUESTED** [review]
There's some issue with the formatting of the the table in AboutUs.md. Could you make the change?

File: docs/AboutUs.md (hunk 1 above)
Comment (zoranabc201) [reply docs/AboutUs.md:8]: Fixed in new PR.

**Pull request closed without merging**